src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/authorization/UserMapper.kt

```diff
@@ -20,6 +17,7 @@ class JwtAuthUserMapper {
     val authSource = authentication.token.getClaimAsString("auth_source")
       ?: throw ServerWebInputException("no 'auth_source' claim in authentication token")
 
+    // should we persist the user here??
     return AuthUser(id = userID, authSource = authSource, userName = userName)
```

Comment:
Unless we want to cache everyone, I'd say no

Reply:
thanks - yeh i agree. as i was writing this i went down a path of thinking we needed users to be persisted for scope mappings, but of course we don't. so this comment should be removed.